datasets/si/conflicts/crawler.py

```diff
@@ -1,57 +1,91 @@
+            new_key = old_key
+        result[new_key] = value
+    return result
+
+
+def crawl_entity(context: Context, record: Dict[str, Any]):
+    subject_name = record.pop("entity_name")
+    registration_number = record.pop("entity_reg_number")
+    country = record.pop("country")
+    start_date = record.pop("start_date")
+    end_date = record.pop("end_date")
+    org_name = record.pop("org_name")
+    org_internal_id = record.pop("org_internal_id")
+
+    # There are some records with no subject name or registration number
+    if not subject_name and not registration_number:
         return
 
+    legal_entity = context.make("LegalEntity")
     legal_entity.id = context.make_id(registration_number or subject_name)
     legal_entity.add("name", subject_name)
     legal_entity.add("registrationNumber", registration_number)
-    legal_entity.add("taxNumber", record.pop("subjekt_davcna"))
+    legal_entity.add("taxNumber", record.pop("entity_tax_number"))
     legal_entity.add("topics", "debarment")
-    legal_entity.add("country", "si")
-
-    organization_name = record.pop("org_naziv")
-    organization_number = record.pop("org_sifrapu")
-
-    if registration_number and organization_number:
+    legal_entity.add("country", country)
+    legal_entity.add(
+        "program",
+        RESTRICTION_NOTE.format(
+            organization=org_name,
+            subject=subject_name,
+            start_date=start_date,
+            until_clause=f"until {end_date}" if end_date else "until further notice",
+        ),
+    )
+    if registration_number and org_internal_id:
         legal_entity.add(
             "sourceUrl",
-            f"https://erar.si/transakcija/placnik/{organization_number}/prejemnik/{registration_number}",
+            f"https://registri.kpk-rs.si/registri/omejitve_poslovanja/seznam/#3={org_internal_id}&8={registration_number}",
         )
 
-    start_date = record.pop("od")
-    end_date = record.pop("do")
-    if end_date.startswith("9999"):
-        end_date = "cancellation"
-    else:
-        end_date = f"to {end_date}"
-
-    legal_entity.add(
-        "program",
-        f"{organization_name} is restricted from procurement from {subject_name} from {start_date} until {end_date} due to ownership or management role of a public official in {organization_name} or their family member. This is a preventative restriction by Komisija za preprečevanje korupcije and implies no wrongdoing.",
+    address = h.make_address(
+        context,
+        street=record.pop("entity_address"),
+        place=record.pop("entity_place"),
+        country=country,
     )
+    h.copy_address(legal_entity, address)
 
-    context.emit(legal_entity)
-    context.audit_data(record, ignore=["omejitev_do", "st_transakcij"])
-
+    sanction = h.make_sanction(context, legal_entity)
+    h.apply_date(sanction, "startDate", start_date)
+    h.apply_date(sanction, "endDate", end_date)
```

Comment:
let's not make Sanction entities for these - this dataset should be handled a bit carefully because it's preventative, not punitive. While we use Sanction entities to describe other debarments, we know some users sometimes over-react when they see them and these are merely public officials and SOE employees probably doing their job.

Reply:
Perhaps add a comment in the code explaining exactly this decision for a year from now when we've forgotten